frontend/src/admin/__tests__/CreateUserPageContents.test.jsx

```diff
@@ -45,46 +35,33 @@ describe("<CreateUserPageContents/> class component", () => {
 
   describe("render() returns a component that", () => {
     it("contains one <CreateUserFormConsole/> component with handleSubmit passed as prop", () => {
-      expect(wrapper.find(CreateUserFormConsole)).toHaveLength(1);
-
-      const formConsoleProps = wrapper.find(CreateUserFormConsole).props();
-
-      expect(formConsoleProps.handleSubmit).toBe(
-        wrapper.instance().handleSubmit
+      const form = wrapper.find(FormConsole);
+      expect(form).toHaveLength(1);
+
+      const expected = {
+        handleSubmit: wrapper.instance().handleSubmit,
+        buttonName: "Create",
+        isValidate: true,
+        passwordError: { upperbound: 5, lowerbound: 0 },
+        passwordInputProps: { maxLength: 20, minLength: 5 },
+        passwordHelperText: "Password must be between 5 to 20 characters"
+      };
+
+      const formConsoleProps = form.props();
+
+      expect(formConsoleProps.handleSubmit).toBe(expected.handleSubmit);
+      expect(formConsoleProps.buttonName).toEqual(expected.buttonName);
+      expect(formConsoleProps.isValidate).toBe(expected.isValidate);
+      expect(formConsoleProps.passwordError).toEqual(expected.passwordError);
+      expect(formConsoleProps.passwordInputProps).toEqual(
+        expected.passwordInputProps
       );
```

Comment:
Shorten this to `expect(formConsoleProps).toMatchObject(expected)`

Reply:
8659c3c